pkgs/development/tools/misc/catppuccin-catwalk/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, fetchPypi
+, python3
+,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "catppuccin-catwalk";
```

Comment:
The project is https://github.com/catppuccin/toolbox, why naming it `catwalk`?

Reply:
`catwalk` is one of the `toolbox` packages.
https://github.com/catppuccin/toolbox#-catwalk

Reply:
Thanks!